tests/basic/kubevirt_test.go

```diff
@@ -827,3 +830,86 @@ var _ = Describe("{KubeVirtPvcAndPoolExpandWithAutopilot}", func() {
 		}
 	})
 })
+
+var _ = Describe("{RebootRootDiskAttachedNode}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("RebootRootDiskAttachedNode", "Reboot the node where VMs root disk is attached", nil, 0)
+		DeployVMTemplatesAndValidate()
+	})
+	var appCtxs []*scheduler.Context
+
+	itLog := "Reboot node where Kubevirt VMs root disk is attached"
+	It(itLog, func() {
+		appList := Inst().AppList
+		defer func() {
+			Inst().AppList = appList
+		}()
+		Inst().AppList = []string{"kubevirt-cirros-live-migration", "kubevirt-windows-vm"}
+		stepLog := "schedule a kubevirtVM"
+		Step(stepLog, func() {
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				log.Infof("RK=> Deploying iteration: %d", i+1)
+				appCtxs = append(appCtxs, ScheduleApplications("reboot")...)
```

Comment:
does this work with scale factor > 1? Just wondering if ScheduleApplications will assign a unique name during each iteration. We are not passing the iteration number to the function.

Reply:
Schedule applications uses a timestamp suffix for namespaces, that way it works with scale factor > 1